txpool/grpc_server.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/ledgerwatch/erigon-lib/chain"
 	"github.com/ledgerwatch/erigon-lib/gointerfaces"
 	txpool_proto "github.com/ledgerwatch/erigon-lib/gointerfaces/txpool"
+	types "github.com/ledgerwatch/erigon-lib/gointerfaces/types"
```

Comment:
This is not required, there is already `types2`

Reply:
I'll fix that